model/src/main/java/org/cbioportal/model/MolecularProfile.java

```diff
@@ -120,7 +122,8 @@ public void setDescription(String description) {
     public Boolean getShowProfileInAnalysisTab() {
         // TODO: remove this logic once the data is fixed (when all the fusions are migrated to structural variants)
         return showProfileInAnalysisTab
-                || (getMolecularAlterationType().equals(MolecularAlterationType.STRUCTURAL_VARIANT)
+                || ((molecularAlterationType.equals(MolecularAlterationType.STRUCTURAL_VARIANT)
+                        || molecularAlterationType.equals(MolecularAlterationType.MUTATION_EXTENDED))
```

Comment:
Are the parenthesis correct here?  Should && only apply to datatype FUSION and MUTATION_EXTENDED?

Reply:
the logic before was SV && FUSION and since our case is mutations && FUSION I decided to consolidate the logic a little

Reply:
Oh, the datatype is FUSION for SV alteration type?  Is that always true?